core/modules/czar/python/app.py

```diff
@@ -379,14 +379,14 @@ def _computeHash(self, bytes):
     def _prepareForExec(self):
         """Prepare data structures and objects for query execution"""
         self.hints = self.context.conditions.copy() # make a copy
-        dbContext = self.hints.get("db", "")
+        db_context = self.hints.get("db", "")
 
         logger.dbg("Setting sessionId")
-        self.sessionId = self.context.uqFactory.newUserQuery(self.queryStr,
-                                                             dbContext,
-                                                             self._resultName)
-        errorMsg = UserQuery_getQueryProcessingError(self.sessionId)
-        if errorMsg: raise ParseError(errorMsg)
+        self.sessionId, self.proxyOrderBy = self.context.uqFactory.newUserQuery(self.queryStr,
+                                                                                db_context,
+                                                                                self._resultName)
+        error_msg = UserQuery_getQueryProcessingError(self.sessionId)
+        if error_msg: raise ParseError(error_msg)
```

Comment:
Why did you rename them? Keep it as it was for consistency with the rest of the file.